core/src/test/java/org/apache/accumulo/core/constraints/VisibilityConstraintTest.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.accumulo.core.security;
```

Comment:
core.security is API. This is a breaking API change.

Reply:
I moved the test to the package of the class its testing. I didn't think Tests where considered API?

Reply:
My mistake. I overlooked that this was in the test sources.